forge-ai/src/main/java/forge/ai/CreatureEvaluator.java

```diff
@@ -59,9 +59,9 @@ public int evaluateCreature(final Card c, final boolean considerPT, final boolea
         }
         if (c.hasKeyword(Keyword.HORSEMANSHIP)) {
             value += addValue(power * 10, "horses");
-        }
-        if (c.hasKeyword("Unblockable")) {
-            value += addValue(power * 10, "unblockable");
+        //} TODO is there a way to parse the CantBlockBy static here?
```

Comment:
I guess you'd need to loop over all the static effects on the card and look for CantBlockBy and inspect its parameters to determine that it's the "unblockable" effect. Might not be the most performance-efficient way of doing it though, but I don't know a better way :/ Maybe someone else would have a better idea?